contracts/contracts/mocks/MockOgvGovernance.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.10;
+
+import { ERC20 } from "openzeppelin-4.6.0/token/ERC20/ERC20.sol";
+
+contract MockOgv is ERC20 {
```

Comment:
name should be changed here as well I think

Reply:
Changed in commit 516a2be9